example/app/config.yml

```diff
@@ -731,15 +732,34 @@ mhs4:
   ecs:
     ssh: true
     keyPairName: mhs
+  launchpad:
+    api: 'https://api.launchpad.nasa.gov/icam/api/sm/v1'
+    passphrase: '{{LAUNCHPAD_PASSPHRASE}}'
   apiConfigs:
     backend:
       private: true
       port: 8004
     distribution:
       private: true
-      port: 7004
+      port: 7014
       logApiGatewayToCloudWatch: true
   logToSharedDestination: '{{SHARED_LOG_DESTINATION_ARN}}'
+  cmr:
+    oauthProvider: launchpad
+    username: CmrUserNotUsed
+    password: CmrPasswordNotUsed
+  oauth:
+    provider: launchpad
+    userGroup: GSFC-Cumulus-Dev
+  # oauth:
+  #   provider: earthdatalogin
+  #   userGroup: n/a
```

Comment:
You will need this for the API Launchpad login stuff

Reply:
I'm using launchpad above?  I just didn't delete those commented out lines.  Is that what you mean?

Reply:
oh yes. right.